docs/content/fantomas-web-components.js

```diff
@@ -0,0 +1,78 @@
+import {html, css, LitElement} from 'https://cdn.skypack.dev/lit';
+export class GreenCategory extends LitElement {
+  
+    static properties = {
+      tooltip: {type: String},
+    };
+
+    createRenderRoot() {
+      return this;
+    } 
+    constructor() {
+      super();
+      this.tooltip = 'This setting is good to use';
+    }
+  
+    render() {
+      return html`<i class="bi bi-check-circle-fill green-recommendation" data-bs-toggle="tooltip" data-bs-custom-class="green-tooltip" data-bs-title="${this.tooltip}"></i>`;
+    }
+  }
+export class OrangeCategory extends LitElement {
```

Comment:
Use one component that takes some settings to combine Green, Orange and Red. The G-Research component can be its own components as it is slightly different.